js/admin/aioseop-admin-functions.js

```diff
@@ -0,0 +1,112 @@
+/**
```

Comment:
Tabs instead of spaces for indenting.

Reply:
@EkoJR fortunately Visual Studio Code makes it really easy.

![image](https://user-images.githubusercontent.com/18156697/69529445-95ab3280-0f70-11ea-82ef-e616e9f691e5.png)